test/suite/test_import01.py

```diff
@@ -77,15 +77,36 @@ def check(self, uri, keys, values):
     # The ID and checkpoint information can be different between configs, remove it. Everything else
     # should be the same.
     def config_compare(self, aconf, bconf):
-        a = re.sub('id=\d+,?', '', aconf)
-        a = (re.sub('\w+=\(.*?\)+,?', '', a).strip(',').split(',') +
-             re.findall('\w+=\(.*?\)+', a))
-        b = re.sub('id=\d+,?', '', bconf)
-        b = (re.sub('\w+=\(.*?\)+,?', '', b).strip(',').split(',') +
-             re.findall('\w+=\(.*?\)+', b))
-        a = [x for x in a if not x.startswith("checkpoint=")]
-        b = [x for x in b if not x.startswith("checkpoint=")]
-        self.assertTrue(sorted(a) == sorted(b))
+        stripped_aconf = self.strip_subconfigs(aconf)
+        stripped_bconf = self.strip_subconfigs(bconf)
+
+        self.assertTrue(sorted(stripped_aconf) == sorted(stripped_bconf))
+
+    # Strips the subconfigs that are irrelevant to config_compare (id and checkpoints).
+    def strip_subconfigs(self, conf):
```

Comment:
```suggestion
    def strip_subconfig(self, conf):
```

Reply:
Please update the two calls after this change.

Reply:
yep, done. Thanks for the suggestions :)